versions/3.0.3.md

```diff
@@ -2284,7 +2284,7 @@ The following properties are taken directly from the JSON Schema definition and
 - exclusiveMinimum
 - maxLength
 - minLength
-- pattern (This string SHOULD be a valid regular expression, according to the [ECMA 262 regular expression](https://www.ecma-international.org/ecma-262/5.1/#sec-7.8.5) dialect)
+- pattern (This string SHOULD be a valid regular expression, according to the [Ecma-262 Edition 5.1 regular expression](https://www.ecma-international.org/ecma-262/5.1/#sec-7.8.5) dialect)
```

Comment:
`Ecma-` should be `ECMA-`, no?

Reply:
Hm, not sure... https://www.ecma-international.org/ecma-262/5.1/#sec-7.8.5 says:

> This is the HTML rendering of *Ecma-262 Edition 5.1,* The ECMAScript Language Specification.

It looks like ECMA is uppercase mostly when used in 'ECMAScript'.

Reply:
Wow, what a mess of mixed caps! :)

You're right though, based on their usage in that doc, we can basically do what we want. My own personal sensibilities would argue for capitalization, and the header on that page does use capitalization: 
![image](https://user-images.githubusercontent.com/54582/74046252-1203be00-4983-11ea-986b-a4f4ab5fa057.png)

... but then does something else shortly thereafter. My point is definitely a nit, and this shouldn't hold up the acceptance, though if you don't mind using `ECMA` instead, please do, thx.